### PR TITLE
warm and bean + line fix

### PR DESCRIPTION
**Pull request by eip618**
add memes and a new line fix in lumabug command
:fingerguns:

### CODE REVIEW

File: addons/memes.py [hunk 1]
```diff
@@ -37,7 +37,19 @@ def __init__(self, bot):
...
+    @commands.cooldown(rate=1, per=15.0, type=commands.BucketType.channel)
+    async def warm(self, ctx):
+        """Memes."""
+        await self._meme(ctx, "User has been warmed.")
 
+    @commands.command(pass_context=True, hidden=True)
+    @commands.cooldown(rate=1, per=15.0, type=commands.BucketType.channel)
+    async def bean(self, ctx):
+        """Memes."""
+        await self._meme(ctx, ":bean:")
```
Comment (chenzw95) [inline addons/memes.py]: This will not work for the bot; correct syntax should be (for animated emote): `<a:bean:462076812076384257>`

Comment (eip618) [reply addons/memes.py]: thank you, this has been updated

**Comment by eip618** [comment]
added 11.8 > 11.9
pls merj

**Comment by ihaveamac** [comment]
cherry-picked c1bcf6816532386a4f635544ed3dba88b091823a

**Pull request closed without merging**